src/main/java/de/cronn/validation_files_diff/helper/PsiElementValidationFileFinder.java

```diff
@@ -0,0 +1,131 @@
+package de.cronn.validation_files_diff.helper;
+
+import com.intellij.openapi.module.Module;
+import com.intellij.openapi.module.ModuleManager;
+import com.intellij.openapi.module.ModuleUtilCore;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.roots.ContentIterator;
+import com.intellij.openapi.vfs.LocalFileSystem;
+import com.intellij.openapi.vfs.VfsUtilCore;
+import com.intellij.openapi.vfs.VirtualFile;
+import com.intellij.openapi.vfs.VirtualFileManager;
+import com.intellij.psi.PsiClass;
+import com.intellij.psi.PsiElement;
+import com.intellij.psi.PsiMethod;
+import de.cronn.validation_files_diff.JoiningStrategy;
+import de.cronn.validation_files_diff.ValidationDiffProjectOptionsProvider;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Stream;
+
+public class PsiElementValidationFileFinder {
```

Comment:
Class 'PsiElementValidationFileFinder' with only 'private' constructors should be declared 'final'

Reply:
Done 4c2aa5ace422f270accd0ad44cabd54d43561f8d